### DISCUSSION TITLE
Multiple THS01 on TH ELite/Pow/Origin

### DISCUSSION THREAD

**Discussion by titou4307** [arendst/Tasmota#16674] [Q&A]
Hello
I would like to connect more than one sensor THS01 (temp + humidity) on a Sonoff TH (flashed on Tasmota) ....

When I connect 2 or 3 sensor all the data are NULL
When I connect only one sensor all is OK (Temp, Humidity.....)

I have understand multiple sensors are possible if the sensor is DS18B20
- I have a TH with 3 sensor plugged between them (all the wire red, all the wire yellow, all the wire black)
- The sensors are named with the serial number...

But impossible to have same result with the THS01 SonOff

Is there an option to have the multiple sensors ?

**Comment by arendst** [comment]
Yes with DS18x20. Nope with all others (unless you hardware mod the device and add I2C support).

The THS01 is not a DS18B20 but a iTead single wire sensor mimicking a DHT22. The DHT22/SI7021 driver does not support multiple sensors on one GPIO.

**Reply by titou4307** [reply]
Hi

My SonOff are all flashed with tasmota, I not keep the firmware Itead.....

I understand the protocol of communication different (but not more.....sorry)

It not possible to connect a sensor on the sensor THS with the J2 port ?? 

![THS01-a](https://user-images.githubusercontent.com/56826797/192549318-eb7376dd-6402-4ec3-93f9-8b7c2bd5dc72.jpg)

![THS01-b](https://user-images.githubusercontent.com/56826797/192547492-9f20ef9f-daf3-49e1-8c05-fb53a87e8095.jpg)

Or another solution is not to code the TH Origin with the sensor SI7021 for have one sensor with temp and humidity and connect into the J2 port (of SI7021 THS01) another sensor like DS18B20 ???
